### ISSUE TITLE
Weird memory usage while runnig .gif file

### ISSUE THREAD

**Issue by Lawa9** [IdolKeg/AnimaPaper#9]
Tested with v0.3 & 0.3.2dev
When I load a ~400MB .mp4 (720p) and let it play I have ~50MB memory usage.
When I load a ~40MB .gif (720p) and let it play I have ~500MB memory usage.

What is going on???

**Comment by jorensanbar** [comment]
Decode a GIF need huge memory, you can see in your browser. Use CloudConvert and convert the GIF into a MP4.

**Comment by IdolKeg** [comment]
I have to agree with jorensanbar. Gifs are working but they are memory heavy.
